script.js

```diff
@@ -19,26 +19,73 @@ const quizData = [
     ],
     correctAnswer: 3,
   },
+  {
+    question: 'What data type represents either TRUE or FALSE values?',
+    choices: ['integer', 'event', 'boolean', 'condition'],
+    correctAnswer: 2,
+  },
+  {
+    question:
+      'What is a block of code called that is used to perform a specific task?',
+    choices: ['variable', 'declaration', 'string', 'function'],
+    correctAnswer: 3,
+  },
+  {
+    question:
+      'What element is used to store multiple values in a single variable?',
+    choices: ['function', 'array', 'variable', 'string'],
+    correctAnswer: 1,
+  },
+  {
+    question: 'Which statement is used to end or exit a loop?',
+    choices: ['break', 'close', 'stop', 'quit'],
+    correctAnswer: 0,
+  },
+
+  {
+    question: 'What element is used to store and manipulate text?',
+    choices: ['loop', 'string', 'array', 'function'],
+    correctAnswer: 1,
+  },
+  {
+    question: 'Which event occurs when the user clicks on an HTML element?',
+    choices: ['onmouseclick', 'onmouseover', 'onclick', 'onchange'],
+    correctAnswer: 2,
+  },
+  {
+    question: 'How do you create a function?',
+    choices: [
+      'function = myFunction()',
+      'let function = myFunction',
+      'function:myFunction()',
+      'function myFunction()',
+    ],
+    correctAnswer: 3,
+  },
 ];
 
+const containerElement = document.getElementsByClassName('container');
 const questionContainer = document.getElementById('question-container');
+const questionCounter = document.getElementById('question-counter');
 const questionElement = document.getElementById('question');
 const choicesElement = document.getElementById('choices');
 const submitButton = document.getElementById('submit-btn');
 const resultElement = document.getElementById('result');
 
 let currentQuestion = 0;
 let score = 0;
+let questionNumber = 1;
 const wrongAnswers = [];
 
 function loadQuestion() {
   const { question, choices } = quizData[currentQuestion];
-  questionElement.textContent = question;
+  questionElement.innerHTML = question;
+  questionCounter.innerHTML = `Question ${questionNumber} of ${quizData.length}`;
```

Comment:
Instead of introducing a new variable for `questionNumber`, you can use the same `currentQuestion` variable here for displaying the question number.
```suggestion
  questionCounter.innerHTML = `Question ${currentQuestion + 1} of ${quizData.length}`;
```